app/build.gradle

```
@@ -20,7 +20,7 @@ android {
 
     buildTypes {
         release {
-            minifyEnabled true
+            minifyEnabled false
```

This value needs to be true to allow code shrinking
```suggestion
            minifyEnabled true
```